src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java

```diff
@@ -284,6 +294,12 @@ public void clickSuModificaPEC() {
         this.element(modificaButtonBy).click();
     }
 
+    public void clickSuAnnullaModificaPEC() {
+        By annullaButtonBy = By.xpath("//p[contains(text(),'PEC associata')]/following-sibling::div/div/button[contains(text(),'Annulla')]");
+        this.getWebDriverWait(30).withMessage("Non si riesce a cliccare sul bottone modifica PEC").until(ExpectedConditions.elementToBeClickable(annullaButtonBy));
+        this.element(annullaButtonBy).click();
+    }
+
```

Comment:
Check https://github.com/pagopa/pn-frontend-e2e/pull/130

Reply:
There is already the method above.

Reply:
questo e click su Annulla